pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -252,6 +252,7 @@ const (
 	RequirementStatusReasonPresent             StatusReason = "Present"
 	RequirementStatusReasonNotPresent          StatusReason = "NotPresent"
 	RequirementStatusReasonPresentNotSatisfied StatusReason = "PresentNotSatisfied"
+	RequirementStatusReasonNotAvailable        StatusReason = "PresentNotAvailable"
```

Comment:
I think a comment here for what exactly not available means would be good - "set when when CRD status condition is missing established" or something.